apps/chat/src/store/conversations/conversations.epics.ts

```diff
@@ -2291,6 +2293,23 @@ const uploadConversationsWithFoldersEpic: AppEpic = (action$) =>
                 conversations: conversations,
               }),
             ),
+            iif(
+              () => !!payload.selectFirst,
+              concat(
+                of(
+                  ConversationsActions.selectConversations({
+                    conversationIds: [conversations[0].id],
+                  }),
+                ),
+                of(
+                  UIActions.openFolder({
+                    featureType: FeatureType.Chat,
+                    id: conversations[0].folderId,
+                  }),
+                ),
```

Comment:
Did you check that `conversations.length > 0`?
```suggestion
            iif(
              () => !!payload.selectFirst && conversations.length > 0,
              concat(
                of(
                  ConversationsActions.selectConversations({
                    conversationIds: [conversations[0].id],
                  }),
                ),
                of(
                  UIActions.openFolder({
                    featureType: FeatureType.Chat,
                    id: conversations[0].folderId,
                  }),
                ),
```